akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -43,6 +45,8 @@ private object ReplicatedDataSerializer {
      */
     def getKey(entry: A): Any
     final def compare(x: A, y: A): Int = compareKeys(getKey(x), getKey(y))
+
+    @silent
     private final def compareKeys(t1: Any, t2: Any): Int = (t1, t2) match {
       case (k1: String, k2: String)             => k1.compareTo(k2)
       case (k1: String, k2)                     => -1
```

Comment:
that could be `case (_: String, _)`, but maybe it's just an experiment to verify that `silent` works?

Reply:
It could be, but in this case that seemed to hurt readability a bit (when we do it for all these cases) - so that's why it seemed like a good candidate to try out `@silent` indeed